### ISSUE TITLE
Allow searching Node Tree in Editor Feature Profiles

### ISSUE THREAD

**Issue by leoossa** [godotengine/godot-proposals#6328]
### Describe the project you are working on

A 2D UI App (just a app with UI Controls)

### Describe the problem or limitation you are having in your project

I want to setup up Editor Feature Profile by removing all 3D features / Nodes that I won't use. Now I have to go one-by-one and turn them off by scrolling, reading all stuff on the list, and then turning off ones I don't need.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

I would love a simple search bar (fuzzy search) just like 'Filter Nodes' in a Scene Panel. Then it would be easy to filter by typing '3D' and filtering all the nodes that have '3D' in their names.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

Just like nodes filter in a Scene panel

### If this enhancement will not be used often, can it be worked around with a few lines of script?

Probably could be a plugin, but I think it would be use often by people who want to use feature of 'Editor Profiles'

### Is there a reason why this should be core and not an add-on in the asset library?

Yes, it's very very common usage and strictly bounded to the 'Editor Profiles' feature.

**Comment by Calinou** [comment]
You don't need to uncheck every 3D node individually. Uncheck Node3D and all its children will also be disabled:

![image](https://user-images.githubusercontent.com/180032/220202646-45d65b07-f943-43b4-b96e-72d09072bce1.png)

**Comment by leoossa** [comment]
@Calinou  that's true but not entirely. For example you have nodes like World3D, Curve3D and others that probably can be disabled as well for GUI app.
